src/components/Details.jsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable linebreak-style */
+import React, { useEffect } from 'react';
+import './styles/Crypto.css';
+import { Link, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowLeft, FaMicrophoneAlt } from 'react-icons/fa';
+import { AiFillSetting } from 'react-icons/ai';
+import { searchCryptos } from '../Redux/Crypto';
+import globe from './assets/globe.png';
+
+function Details() {
+  const animes = useSelector((state) => state.animes);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(searchCryptos());
+  }, [dispatch]);
+  const { name } = useParams();
+  const index = animes.findIndex((data) => data.name === name);
+
+  return (
+    <div className="details-container" data-testid="detailsdiv">
+      <div className="details-nav nav-bar">
+        <Link to="/">
+          <span className="arrow">
+            <FaArrowLeft />
+          </span>
+        </Link>
+        <p className="nav-name">{animes[index].name}</p>
+        <div className="nav-right">
+          <div className="icons">
+            <FaMicrophoneAlt />
+            <AiFillSetting />
+          </div>
+        </div>
+      </div>
+      <div className="imagedetails">
+        <h1>{animes[index].symbol}</h1>
+        <div className="title-details">
+          <h2>
+            #
+            {animes[index].rank}
```

Comment:
- [ ] When I refreshed the details page, all the content seems to disappear, Kindly fix it 👍🏽 . 

I would recommend you save the specific data in the local storage, so that on refresh, you can use the stored data in the local storage to populate the UI 😄